domains/interjadify.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Jadify-eu",
+           "email": "jadifyeu@gmail.com",
+           "discord": "753179409682399332"
+        },
+    
+        "record": {
+            "TXT": "will be changed"
```

Comment:
This does nothing

Reply:
I cannot find the file with that text ( my browser crashes when opening the domains folder